build/websocket-scenarios.yml

```
@@ -16,8 +16,14 @@ parameters:
   type: object
   default: 
 
-  - displayName: Echo
-    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --property scenario=echo
+  - displayName: Echo-10000
+    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --variable size=10000--property scenario=echo
+  - displayName: Echo-100
+    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --variable size=100 --property scenario=echo
+  - displayName: Echo-Compression-100
+    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --variable size=100 --variable compress=true --property scenario=echo --application.buildArguments \"/p:UseCompression=true\"
```

```suggestion
    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --variable size=100 --variable compress=true --property scenario=echo --application.buildArguments \"/p:UseCompression=true\" --property size=100 --property compress=true
```

Note that `\"` is only used because this string is concatenated in the json payload later. In Yaml we could just use `"` directly here otherwise.